binstar_client/__init__.py

```diff
@@ -369,7 +369,8 @@ def remove_release(self, username, package_name, version):
         self._check_response(res, [201])
         return
 
-    def add_release(self, login, package_name, version, requirements, announce, description):
+    def add_release(self, login, package_name, version, requirements, announce,
+                    description, icon=None):
```

Comment:
That is done in another place (I used the upload thumbnail logic which incorporates that part)